cinder/volume/drivers/nexenta/ns5/jsonrpc.py

```diff
@@ -149,7 +149,31 @@ def handle_failover(self):
             self.__proxy.__init__(
                 host, self.__proxy.port, self.__proxy.user,
                 self.__proxy.password, self.__proxy.use_https,
-                self.__proxy.verify)
+                self.__proxy.pool, self.__proxy.verify)
+            url = self.get_full_url('rsf/clusters')
+            response = self.__proxy.session.get(
+                url, verify=self.__proxy.verify)
+            content = response.json() if response.content else None
+            cluster_name = content['data'][0]['clusterName']
+            for node in content['data'][0]['nodes']:
+                if node['ipAddress'] == self.__proxy.host:
+                    node_name = node['machineName']
+            counter = 0
+            while counter < 24:
+                counter += 1
+                url = self.get_full_url(
+                    'rsf/clusters/%s/services' % cluster_name)
+                response = self.__proxy.session.get(url)
+                content = response.json() if response.content else None
+                for service in content['data']:
```

Comment:
Not good. This 'for' crashes the program if 'content' is None that can happen in previous line